### DISCUSSION TITLE
Example 21: Row Detail View

### DISCUSSION THREAD

**Discussion by rgrosser71** [ghiscoding/Angular-Slickgrid#1078] [Q&A] [answered]
Hi,
has somone made an example with a Angular-Slickgrid as Detail-Data?
I've tried, but it seems that it get some Problems,
because editing at master not work, and
collapse detail via Service-call dont work,
bacause angularGrid.extensions.rowDetailView not exists,
nor angularGrid.extensionService.getExtensionInstanceByName(ExtensionName.rowDetailView).

Best regards
Robert

**Comment by ghiscoding** [comment]
The Row Detail feature was created to show well... row detail, it wasn't created to show a grid in a grid, that won't work and is not supported (I know there are a few issues when trying to do that). For that use case you should use 2 separate grids (master / child grids).

**Reply by ghiscoding** [reply, marked as answer]
You can now add a grid inside a Row Detail starting from version [v8.13.1](https://github.com/ghiscoding/Angular-Slickgrid/releases/tag/v8.13.1), you can see a new [Example 45](https://ghiscoding.github.io/Angular-Slickgrid/#/rowdetail-innergrid) demoing this new feature
